src/Components/Board/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+import Cell from "../Cell";
+import "./Board.css";
+
+const Board = () => {
+  const [cellsData, setCellsData] = useState(
+    Array.from({ length: 9 }, () => {
+      return { id: crypto.randomUUID(), value: "" };
+    })
+  );
+  const [isFirstPlayer, setIsFirstPlayer] = useState(true);
+  const [playersScore, setPlayersScore] = useState({ player1: 0, player2: 0 });
+  const [roundCount, setRoundCount] = useState(1);
+  const LINES = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+  ];
+
+  useEffect(() => {
+    for (let i = 0; i < LINES.length; i++) {
+      let indexes = LINES[i];
+      let [index1, index2, index3] = indexes;
```

Comment:
The correct term is `indices`.
```suggestion
      let [index1, index2, index3] = indices;
```